pilot/pkg/features/pilot.go

```diff
@@ -113,6 +113,12 @@ var (
 			" to unhealthy/non-ready hosts even if the percentage of healthy hosts fall below minimum health percentage(panic threshold).",
 	).Get())
 
+	EnablePersistentSessionFilter = env.Register(
+		"PILOT_ENABLE_PERSISTENT_SESSION_FILTER",
+		false,
```

Comment:
This was added because @howardjohn had a concern about adding EmptySessionFilter. I am OK to default this to true and people does not have to double opt-in. IMO, node metadata is not needed for this as we already have service level annotation driving this.